locales/et-EE.json

```diff
@@ -0,0 +1,2151 @@
+{
```

Comment:
Just for my understanding, is this file download as-is from transifex?

Reply:
The file is downloaded as-is except for one manual step to set these two fields:
```
  "posix_locale": "et_EE.UTF-8",
  "aliases": [
    "et"
  ],
```